oak_functions_launcher/src/server.rs

```diff
@@ -40,7 +40,7 @@ impl oak_remote_attestation_noninteractive::proto::streaming_session_server::Str
         &self,
         request: Request<Streaming<RequestWrapper>>,
     ) -> Result<Response<Self::StreamStream>, tonic::Status> {
-        log::debug!("handling client request");
+        log::info!("handling client request");
```

Comment:
Do we want that as an info log?

Reply:
Yeah I think that's something we expect to see even if not actively debugging, and it's once per request, so I think info is more appropriate.